thirdeye-ui/src/app/pages/welcome-page/landing/landing-page.component.tsx

```diff
@@ -136,9 +141,13 @@ export const WelcomeLandingPage: FunctionComponent = () => {
 
                         <Box clone pb={3} pt={2} textAlign="center">
                             <Typography color="secondary" variant="body2">
```

Comment:
Make the rest of the text not gray

Reply:
Updated

![Screenshot_187](https://user-images.githubusercontent.com/20799363/222080920-ab914ab7-4cd4-4832-aa80-82486fa7e686.png)